### DISCUSSION TITLE
Where is branch releases/ubuntu20/20230903 ??

### DISCUSSION THREAD

**Discussion by KendoChris** [actions/virtual-environments#8269] [General questions]
We were able to clone the above last week, we are being told this is the current branch, but it seems to have disappeared.

warning: Could not find remote branch releases/ubuntu20/20230903 to clone.
fatal: Remote branch releases/ubuntu20/20230903 not found in upstream origin

**Comment by appatalks** [comment]
:wave: @2E0MHQ!

I was able to dig into this and found this [branch was deleted](https://github.com/actions/runner-images/activity?ref=releases/ubuntu20/20230903)  just last week. The closest tag appears to be an updated release [ubuntu20/20230903.1](https://github.com/actions/runner-images/releases/tag/ubuntu20/20230903.1), with the current latest branch being [releases/ubuntu20/20230911](https://github.com/actions/runner-images/tree/releases/ubuntu20/20230911)

I hope this helps!
-- appatalks

**Comment by KendoChris** [comment]
Hi

I have just tried out pipeline again and it’s now seeing the version 20230911 and was able to clone it. I guess something in Git was still indicating the deleted version was still latest.

Anyway it is working now.

Thanks

Chris

**Comment by vpolikarpov-akvelon** [comment]
Hi @2E0MHQ.

Please don't use release branch or tags here. We create them only to mark waypoints in continuous process of image updating. They are not intended for use by end users. There are many software components in the image that we don't pin to specific versions. The latest version is installed every time image is built. This is done to ensure that the software is always up to date. The list of software and it's versions is generated after the image is built. So even if you use the same release branch or tag, you may get different versions of software. From the other hand, image generation scripts become obsolete very quickly because of the changes in the software. So main branch is the stabelest one. Please use it instead of release branches or tags.
